component/src/main/java/org/wso2/extension/siddhi/execution/reorder/utils/WindowCoverage.java

```diff
@@ -0,0 +1,128 @@
+package org.wso2.extension.siddhi.execution.reorder.utils;
```

Comment:
Do we required to have the license?

Reply:
We need to have license header. Fixed in #38